apps/api/src/app.module.ts

```diff
@@ -1,19 +1,28 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { BullModule } from '@nestjs/bull';
+import { ScheduleModule } from '@nestjs/schedule';
+import { SummaryModule } from './modules/summary/summary.module';
 import { DatabaseModule } from './database/database.module';
 import { GraphQLModule } from '@nestjs/graphql';
 import { join } from 'path';
 import { ApolloDriver, ApolloDriverConfig } from '@nestjs/apollo';
 import { ConfigService } from '@nestjs/config';
-import { SummaryModule } from './modules/summary/summary.module';
-;
 
 const configService = new ConfigService();
 @Module({
   imports: [
     DatabaseModule,
     SummaryModule,
+    BullModule.forRoot({
+      redis: {
+        host: 'localhost',
```

Comment:
Get these from .env like in osmoX

Reply:
Updated the app.module and added variables in .env file